content/en/docs/collector/Install/_index.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Please rename this directory to be all lower case: `install` instead of `Install`.